pkg/Microsoft.NETCore.Platforms/runtimeGroups.props

```diff
@@ -141,11 +141,8 @@
       <Architectures>x64;x86;arm;arm64</Architectures>
       <Versions>7;8;81;10</Versions>
     </RuntimeGroup>
-
     <!-- add corert to all RIDs -->
-    <RuntimeGroupWithQualifiers Include="@(RuntimeGroup)">
-      <AdditionalQualifiers>%(AdditionalQualifiers);corert</AdditionalQualifiers>
-    </RuntimeGroupWithQualifiers>
+    <RuntimeGroupWithQualifiers Include="@(RuntimeGroup)" />
```

Comment:
We could remove the RuntimeGroupWithQualifiers item entirely and just use RuntimeGroup.

Reply:
It looks like something gets added further down. Since I don't have insight into how this gets used, I'm not going to do further refactoring in this change.